src/net.h

```diff
@@ -234,7 +235,7 @@ void RemoveLocal(const CService& addr);
 bool SeenLocal(const CService& addr);
 bool IsLocal(const CService& addr);
 bool GetLocal(CService &addr, const CNetAddr *paddrPeer = nullptr);
-CAddress GetLocalAddress(const CNetAddr *paddrPeer, ServiceFlags nLocalServices);
+CAddress GetLocalAddress(const CNetAddr *paddrPeer);
```

Comment:
9d9a6677c483772fe636b61480dfdde7d4586073 (nit): Could run clang-format on this line while touching it?